### PR TITLE
Add assignee input

### PR DESCRIPTION
**Pull request by lee-dohm**
Adds the ability to set the assignee by input if you need to use Action logic to determine to whom the issue should be assigned.

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -7,6 +7,8 @@ branding:
   icon: alert-circle
   color: gray-dark
 inputs:
+  assignee:
```
Comment (JasonEtco) [inline action.yml]: I wonder if this should be `assignees`? It'd be consistent with the API parameter at least.

Comment (lee-dohm) [reply action.yml]: Sounds good. Made the change 👍

**Comment by lee-dohm** [comment]
Not sure why CI didn't run. Perhaps because it's from a fork?

**Comment by JasonEtco** [comment]
> Perhaps because it's from a fork?

That definitely shouldn't happen, but I'll figure that out separately. For now, thank you for the addition 🎉 👏

**Pull request merged**